pinc/html_page_common.inc

```diff
@@ -63,6 +63,7 @@ function output_html_header($nameofpage, $extra_args = array(), $show_statsbar =
     $theme_name = $user ? $user->i_theme : "project_gutenberg";
     $css_files = array(
         "$code_url/styles/themes/$theme_name.css",
+        "$code_url/pinc/3rdparty/fontawesome/css/all.min.css",
```

Comment:
Do we need all the different font styles? I'm sure the browser probably only requests the font files for the used style sheets, but may prefetch if it gets bored and we could avoid that if we used just the stylesheets for the fonts we intend to use. I'm specifically wonder about brands:
https://fontawesome.com/how-to-use/on-the-web/setup/hosting-font-awesome-yourself#using-certain-styles

Reply:
I thought it easiest if we include all of the styles so if a developer goes to the Font Awesome website and sees an icon in the Free pack they want to use then it's available. I actually want the brands for pgdp.net so I can remove the PROD hack for the social media icons :D